addons/html_editor/static/tests/table/ui.test.js

```diff
@@ -855,3 +855,85 @@ test("preserve table rows width on move row below operation", async () => {
         </table>`)
     );
 });
+
+test("reset table size to remove custom width", async () => {
+    const { el, editor } = await setupEditor(
+        unformat(`
+        <table style="width: 150px;">
+            <tbody>
+            <tr><td style="width: 100px;" class="a">1[]</td></tr>
+            <tr><td style="width: 50px;" class="b">2</td></tr>
+            </tbody>
+        </table>`)
+    );
+    expect(".o-we-table-menu").toHaveCount(0);
+
+    await hover(el.querySelector("td.a"));
+    await waitFor(".o-we-table-menu");
+    expect("[data-type='row'].o-we-table-menu").toHaveCount(1);
+
+    await click("[data-type='row'].o-we-table-menu");
+    await waitFor(".dropdown-menu");
+    await click(queryOne(".dropdown-menu [name='reset_size']"));
+    expect(getContent(el)).toBe(
+        unformat(`
+        <table>
+            <tbody>
+                <tr><td style="" class="a">1[]</td></tr>
```

Comment:
question : why do we still have an empty style attribute instead of no attribute at all ?

Reply:
Because we remove width or weight but not the rest (it s legacy code). If you want I can add a check if style == "" => remove it ?

Reply:
I mean it's not really the point of this PR and it's certainly not a priority. 
But we might want to automate the cleaning of empty style attribute in the normalize, maybe.
Not need to change anything here anyway. I was just wondering if there was a reason I was not aware of.